File: 03 C C++ Python R and Mojo review/03-3 R Language Review/01 Pointers/02.R
value <- 42
ptr1 <- list(value)  # List containing the value (simulates pointer to value)
ptr2 <- list(ptr1)   # List containing ptr1 (simulates pointer to pointer)
ptr3 <- list(ptr2)   # List containing ptr2 (simulates pointer to pointer to pointer)

# Access the value through multiple levels of references
cat("Value:", ptr3[[1]][[1]][[1]], "\n")  # Output: 42

File: 03 C C++ Python R and Mojo review/03-2 Python Review/01 Pointers/02.py
value = 42
ptr1 = [value]  # List containing the value (simulates pointer to value)
ptr2 = [ptr1]   # List containing ptr1 (simulates pointer to pointer)
ptr3 = [ptr2]   # List containing ptr2 (simulates pointer to pointer to pointer)

# Access the value through multiple levels of references
print(f"Value: {ptr3[0][0][0]}")  # Output: 42